packages/angular/cli/src/analytics/analytics-collector.ts

```diff
@@ -52,8 +53,7 @@ export class AnalyticsCollector {
 
     this.requestParameterStringified = querystring.stringify(requestParameters);
 
-    // Remove the `v` at the beginning.
-    const nodeVersion = process.version.substring(1);
+    const parsedVersion = semver.parse(process.version.slice(1) /* drop leading 'v' */);
```

Comment:
The `semver.parse` will clean up any leading `v`.

```suggestion
    const parsedVersion = semver.parse(process.version);
```